Update get-started-with-wp-scripts.md (#64436)
WordPress/gutenberg: docs/getting-started/devenv/get-started-with-wp-scripts.md

### Enqueuing assets

If you register a block via `register_block_type` the scripts defined in `block.json` will be automatically enqueued:

```php
/**
 * Register the block.
 */
function example_register_block() {

>     // build folder which contains the "block.json" file
> 	register_block_type( __DIR__ . '/build' );

Reviewer: ```suggestion
	// The build folder contains the block.json file.
	register_block_type( __DIR__ . '/build' );
```

}

add_action( 'init', 'example_register_block' );
```

> See full [example](https://github.com/WordPress/block-development-examples/tree/trunk/plugins/minimal-block-ca6eda)

Reviewer: ```suggestion
View the complete [example](https://github.com/WordPress/block-development-examples/tree/trunk/plugins/minimal-block-ca6eda) in context.
```

> This will both register the block on server-side and client-side. More information regarding the registering of a block can be found in [Registration of a block](https://developer.wordpress.org/block-editor/getting-started/fundamentals/registration-of-a-block/)

Reviewer: ```suggestion
Using this approach will register the block on the server. More information regarding the registering of a block can be found in [Registration of a block](https://developer.wordpress.org/block-editor/getting-started/fundamentals/registration-of-a-block/).
```

Using `register_block_type` doesn't register the block in the client, only the server.

> 
> #### Manual enqueuing

Reviewer: ```suggestion
```

I don't think we need another heading here.
